tests/integration/local/invoke/test_integrations_cli.py

```diff
@@ -416,8 +416,8 @@ def test_invoke_returns_expected_results_from_git_function(self):
         process_stdout = stdout.strip()
         self.assertEqual(process_stdout.decode("utf-8"), '"git init passed"')
 
+    # For Windows, this test must run with administrator privilege
```

Comment:
Can you explain why this is?

Reply:
Git layer zip file contains symbolic link. Python uses os.symlink which on newer versions of Windows 10 requires Developer Mode enabled, SeCreateSymbolicLinkPrivilege privilege, or running as Administrator.

Reply:
Added Click based exception.